client/app/configuration/configuration.controller.js

```diff
@@ -37,6 +37,14 @@ angular.module('App').controller(
 
     buildSummitData() {
       this.localeForSummitBanner = this.$translate.use().split('_')[0] === 'fr' ? 'fr' : 'en';
+
+      const subsidiariesWithSummitBanner = ['FR', 'GB', 'DE', 'ES'];
+      this.shouldDisplayBanner = false;
+      return this.User
```

Comment:
So buildSummitData` method returns now a `Promise`. You can easilly do a `$q.all()` or chain both promises you have in `$onInit` method, Right?

But pay attention L19, you already fetch the current user;)

Reply:
I know all of this, it's supposed to be temporary and quickly done